lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -566,6 +568,12 @@ defmodule ExUnit.DocTest do
     extract_tests(lines, "", "", acc, true)
   end
 
+  # Encountered end of fenced code block, store pending test
+  defp extract_tests([{<<fence::3-bytes>> <> _, _} | lines], expr_acc, expected_acc, [test | t], _) when fence in @fences and expr_acc != "" do
```

Comment:
I think it's better to break this up into 2 lines. remember to leave 4 space indentation before `when`